src/scenes/ProgressReports/StatusHistory/WorkflowEventList.tsx

```diff
@@ -37,10 +39,10 @@ export const WorkFlowEventList = ({
         containerType: 'inline-size',
       }}
     >
-      {events.map((event: WorkflowEventFragment, i: number) => (
+      {reversedEvents.map((event: WorkflowEventFragment, i: number) => (
```

Comment:
```suggestion
      {list.map((event) => (
```
The reverse could happen here too. Up to you.
```suggestion
      {list.reverse().map((event) => (
```